src/richie/apps/core/templates/richie/hreflang.html

```diff
@@ -0,0 +1,7 @@
+{% load i18n menu_tags cms_tags %}
+######################################### {{languages}}
```

Comment:
What is the purpose of this line?

Reply:
![image](https://user-images.githubusercontent.com/1427165/119979036-af409680-bfba-11eb-88cc-c3ebdbb98d21.png)